### ISSUE TITLE
Cannot assign to read only property '0' of object '[object Array]'

### ISSUE THREAD

**Issue by jjanderson** [kalkih/mini-graph-card#286]
Hello,

After the upgrade today, I am getting this error on some of my graph's...

Here is a sample of one of the graph's throwing errors:

```
color_thresholds:
  - color: '#00CD00'
    value: 65
  - color: '#98FB98'
    value: 60
  - color: '#FFFF00'
    value: 51
  - color: '#FF9912'
    value: 38
  - color: '#B20000'
    value: 25
entities:
  - entity: sensor.bedroom_iaq_index
    name: Air Quality
hours_to_show: 168
icon: 'mdi:biohazard'
name: Air Quality
type: 'custom:mini-graph-card'
```

**Comment by Poeschl** [comment]
Happening on Home Assistant 0.106.0

**Comment by rickjmatthews** [comment]
Yup - same issue here with 0.106.0

**Comment by Poeschl** [comment]
Updated the card to the latest version via HACS and it's working again.

**Comment by kalkih** [comment]
This has been fixed in the latest version of the card, released a few days back, please see #274 
Update the card, if that doesn't help clear cache, reinstall the card in HACS, double check the resource reference.

Duplicate of #274, #275, #284, #285

**Comment by rickjmatthews** [comment]
Thanks - reinstalling and clearing the browser cache sorted it for me.
